src/views/Orders/Order.vue

```diff
@@ -1,24 +1,28 @@
 <template>
 
-  <div class="cart-box container">
-
-    <h2>Orders</h2>
-    <div v-if="orders">
-
-          <div v-for="itr in this.len" :key="itr">
-
-              <div class="order_class">
-                <h4><router-link v-bind:to="'/order/'+this.orderList[itr-1].pid">Order No : {{itr}}</router-link></h4> 
-                <h4>Total Cost : $ {{totalCost[itr-1]}}</h4>
-                
-              </div>
-
-              <h4>Ordered on : {{orderdate[itr-1]}}</h4>
-
-          </div>
-
+  <div class="container">
+    <div class="row">
+      <div class="col-12 text-center">
+        <h4 class="pt-3">Your Orders</h4>
+      </div>
     </div>
 
+    <div v-if="orders" v-for="itr in len" :key="itr" class="row mt-2 pt-3 justify-content-around">
```

Comment:
iterate over items instead of index.

Reply:
The template as has two arrays which contain info about the same order at same index. Since, we are using both arrays, we have to iterate using the index